integration/buttons.spec.ts

```diff
@@ -6,6 +6,7 @@ fixture("Buttons").page`http://localhost:3000`;
 const Config = "./../src/test/fixtures/sample-config-local.json";
 
 const Title = Selector("h1");
+const Title3 = Selector("h3");
```

Comment:
lets data-testid this maybe, h3 abit too loose. then maybe lose the h1 selector if can